app/src/main/res/drawable/ic_outline_language_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:alpha="0.75" android:height="24dp"
```

Comment:
Remove this icon, actually I don't know how we can implement this feature right now.